package/boot/uboot-kirkwood/Makefile

```diff
@@ -1,16 +1,16 @@
 #
-# Copyright (C) 2010-2014 OpenWrt.org
+# Copyright (C) 2010-2020 OpenWrt.org
```

Comment:
Don't do that
https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#makefile-contents-should-contain